.github/workflows/publish.yml

```
@@ -0,0 +1,20 @@
+name: Publish package
+
+on:
```

yamllint

[warning] truthy value should be one of [false, true]

(truthy)